src/azure-cli/azure/cli/command_modules/synapse/_params.py

```diff
@@ -171,6 +173,11 @@ def load_arguments(self, _):
                    help='The library requirements file.')
         c.argument('force', arg_type=get_three_state_flag(), help='The flag of force operation.')
 
+        # Custom Libraries
+        c.argument('package_action', arg_group='Custom Libraries', arg_type=get_enum_type(['Add', 'Remove']),
+                   help='Package action must be specified when you add or remove a workspace package from a Apache Spark pool.')
+        c.argument('package', arg_group='Custom Libraries', nargs='+', help='List of workspace packages name.')
```

Comment:
If this parameter is not required, you should use `nargs='*'` instead. Or else customer always need to specify `--package` for `az synapse spark pool update`

Reply:
I think users do not need to specify everytime when they run cmd `az synapse spark pool update`?

Below cmd works well
`az synapse spark pool update --resource-group xx --workspace-name xx --name xx`

I only got error when
`az synapse spark pool update --resource-group xx--workspace-name xx --name xx --package-action Remove --package`
argument --package: expected at least one argument